documentation/src/docs/transactions.md

```diff
@@ -1,3 +1,7 @@
+---
+title: Transactions
```

Comment:
Same problem with titles: right side index does not render properly one title ("Pravega Transactions"), which is turn is super large.

Reply:
fixed title sizes and right-side index